### ISSUE TITLE
Alternate Transportation

### ISSUE THREAD

**Issue by daffodilian** [ForgottenGlory/Living-Skyrim-2#300]
**LE/Oldrim mods will be automatically rejected, do not submit them.**

**To submit a mod for consideration, you must attach a copy of your save game where you have defeated Alduin using an unmodified LS installation. Any suggestions without this attached will be automatically rejected. This is not a guarantee that the mod will be included.**

**Mod Name**
Scrolls and Spells of Passage - Teleport to Cities and Towns From Anywhere
**Mod Link**
[Nexus Page](https://www.nexusmods.com/skyrimspecialedition/mods/8881?)

**Why?**
Skyrims fast travel systems are meh at best. Normal fast travel is boring and immersion breaking for me, and carriages are fine to a certain point, but would the Arch-Mage, or Dragonborn really be caught on a carriage? This mod would add a new method of traveling, geared towards later game as they are rather expensive. 

**Additional context**
I had used this mod religiously on XB1 and it had always worked as intended. The tomes and scrolls are added to Innkeeper and Mage merchants leveled lists by script. There is also a scrolls only version, so they have to  be purchased each time, I think this would also make a great goldsink.

**Comment by KNGLogan** [comment]
psst you need a save

**Comment by daffodilian** [comment]
can i use urs

**Comment by TwistedModding** [comment]
No I got dibs

**Comment by LivingSkyrimFanBoy** [comment]
@daffodilian 

Yeah I agree, Skyrim's fast travel system really kills the immersion, it makes things less exciting.

I'm not sure if you've heard of this mod but I highly recommend SWIFT, it adds new teleport shrines around the map to let you travel seamlessly in the game, this could be just what we're looking for. This mod even provides a quest (extra content) to encourage the player to explore these lost sacred shrines.

If you're considering different ideas or alternatives, I think this is definitely it.

**Name:** Skyrim Wayshrines - Immersive Fast Travel
**Link:** https://www.nexusmods.com/skyrimspecialedition/mods/17905

I don't know what the limitations are but maybe we could have both SWIFT and Scrolls and Spells of Passage? That's a question for the developers though.

I personally don't have a save just yet, I just wanted to present this idea to everyone because we all want what's best for LS.

**Comment by jokerstyle00** [comment]
I will say, keep fast travel as an option for those who want it. I could care less about adding alternatives, but it'll be irritating if fast travel is permanently disabled for the people who use it.

**Comment by daffodilian** [comment]
@LivingSkyrimFanBoy I completely forgot about that mod, nice find! 

@jokerstyle00 no worries there, definitely always have the option there.

**Comment by LivingSkyrimFanBoy** [comment]
> I will say, keep fast travel as an option for those who want it. I could care less about adding alternatives, but it'll be irritating if fast travel is permanently disabled for the people who use it.

The best thing about SWIFT is that, there's an option in the MCM menu where you can enable or disable fast travel. So there's plenty of freedom for everyone!
